support/magicleap/Servo2D/scenes/Servo2D.design

```diff
@@ -4,6 +4,7 @@
   <property id="sceneName" value="Servo2D"/>
   <node name="contentPanel" nodeTypeId="lumin.ui.panel">
     <property id="cursorInitialPosition"/>
+    <property id="cursorVisible" value="false"/>
```

Comment:
is this what makes the builtin laser visible? how does setting it to false work?

Reply:
https://mozilla.logbot.info/servo/20190812#c16536817